github_common.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import github3
+
+def repositories_by_topic(github, topic):
+    """
+        Retrieve all commits with a precise topic.
+
+        :param github: the github object
+        :param topic: name of the topic that needs to be match for
+        :return: repository list
+    """
+    repos = []
+    topic_text =  _format_topic(topic)
```

Comment:
no cause 'tuple' object has no attribute 'replace'

Reply:
wait a sec... then if you get a tuple why do you have to split them?
it's because you've declared the  param as multi value which means that you should probably pass it several times. I'd remove the multi option and rely on on commas and do only `[x.strip() for x in topics.split(',') if x.strip()]`